longestString.js

```diff
@@ -0,0 +1,11 @@
+function longestString(array) {
+  if (array.length === 0) {
```

Comment:
A couple things - array.length will return a truthy or falsy value, you can trim this down to (!array.length), HOWEVER, I unless you specifically want to return a null value, I don't really see a need. I think you can wrap lines 6-9 in an if (array.length) block. If the array is empty, it won't break into that block and nothing will happen.